src/terminal_sandbox.mli

```diff
@@ -4,6 +4,6 @@ val create : ?name:string -> ?command:Cmd.spawn -> Sandbox.t -> t
 
 val dispose : t -> unit
 
-val show : t -> unit
+val show : ?preserveFocus:bool -> t -> unit
```

Comment:
Might as well make this argument required.

Reply:
done :)